nntrainer/layers/conv2d_layer.cpp

```diff
@@ -397,6 +399,8 @@ void Conv2DLayer::forwarding(RunLayerContext &context, bool training) {
   filter_kernel.reshape(filter_dim_squeezed);
 
   Tensor &im2col_result = context.getTensor(wt_idx[ConvParams::inter_result]);
+  im2col_result.setZero();
```

Comment:
I wonder if 'im2col_result.setZero()' has association with comment on line from 404 to 407.
If so, should it be moved?

Reply:
right. fixed. Thanks.